src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -5,3 +5,62 @@ title: Working with Robots
 description: How to safely handle and work with robots.
 slug: /guides/hardware/working-with-robots
 ---
+
+## Aligning an Ankle
+
+An alignment of a ankle may need to be done if the foot and the leg of the robot aren't perpendicular when standing. 
+An ankle alignment may also be necessary when the foot has come loose. 
+
+To align an ankle correctly perform the following instructions:
+
+1. On the inner part of the robots leg, unscrew the screws holding the plastic casing to the the ankle and knee servo. 
+Remove the plastic casing, as seen in the following image:
+![leg cover](./images/ankle_leg/leg_cover.jpg 'leg cover')
+
+2. There should be a silver disk, the following photo is a close up:
+![servo close up](./images/ankle_leg/servo_close.jpg 'servo close up')
+There are three small indents on the silver disk, two on one side and one on the other. The single dot should be pointing towards the ground when the robot is standing and the other two dots towards the head.
+These three dots form an arrow, this arrow should be pointing downwards when the robot is standing.
+
+3. Manually move the disk on the servo so it is in the correct orientation 
+
+4. Place the plastic cover back on and replace all the screws. When replacing the screws, one hole must be left without a screw to allow for the servo to move. The hole left without a screw is the largest hole, which can be seen on the right of the following image:
+![screw hole](./images/angle_leg/screw_holes.jpg 'screw hole')
+
+## Replacing Shoulder servo
+
+1. Detatching the arm
+
+    - On the shoulder of the robot, there is a ring of screws front and back. Remove all the screws and the larger centre screw on both sides
```

Comment:
Which larger centre screw? Removing any screw from the centre (the screw the holds the servo horn onto the servo) can cause issues when trying to realign these

Reply:
When disconnecting plastic covers from the servos it is never necessary to remove the large center bolt. Only the smaller bolts in the outer ring need be removed.

Reply:
Yep thanks, I will remove that